configurate-core/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
       <groupId>ninja.leaping.configurate</groupId>
       <artifactId>configurate-parent</artifactId>
-      <version>3.2.1-SNAPSHOT</version>
+      <version>3.3</version>
```

Comment:
The only time the version field should be changed is by me when I'm making a release with maven release plugin, not you.